workflow/react-first-steps/src/store/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux'
+import fetchRandomCocktail from './fetchRandomCocktailReducer'
+
+const reducers = combineReducers({
+    fetchRandomCocktail: fetchRandomCocktail
```

Comment:
you can just pass {
fetchRandomCocktail,
}
check https://alligator.io/js/object-property-shorthand-es6/